tests/test_infer/test_bloom_infer.py

```diff
@@ -27,15 +32,21 @@ def run(test_config):
     sub_model_zoo = model_zoo.get_sub_registry('transformers_bloom_for_causal_lm')
     for name, (model_fn, data_gen_fn, _, _, _) in sub_model_zoo.items():
         orig_model = model_fn()
+        tokenizer = DummyTokenizer()
         orig_model = orig_model.half()
         data = data_gen_fn()
 
         shard_config = ShardConfig(enable_tensor_parallelism=True if test_config['tp_size'] > 1 else False,
                                    inference_only=True)
-        infer_engine = TPInferEngine(orig_model, shard_config, MAX_BATCH_SIZE, MAX_INPUT_LEN, MAX_OUTPUT_LEN)
+        infer_engine = TPInferEngine(orig_model,
+                                     shard_config,
+                                     MAX_BATCH_SIZE,
+                                     MAX_INPUT_LEN,
+                                     MAX_OUTPUT_LEN,
+                                     tokenizer=tokenizer)
 
         generate_kwargs = dict(do_sample=False)
-        outputs = infer_engine.generate(data, **generate_kwargs)
+        outputs = infer_engine.generate(prompt_token_ids=data, **generate_kwargs)
```

Comment:
TypeError: prompt_token_ids type hint as ` Optional[List[List[int]]] = None`, while here data is of type `dict`